action/remove.go

```diff
@@ -18,10 +18,7 @@ var kubeGet kubeGetter = func(m string) string {
 	log.Debug("Getting manifests from %s", m)
 
 	a := []string{"get", "-f", m}
-	out, err := exec.Command("kubectl", a...).CombinedOutput()
-	if err != nil {
-		log.Err("Could not get manifest. %s", err)
-	}
+	out, _ := exec.Command("kubectl", a...).CombinedOutput()
```

Comment:
+1 that it was a misleading error, but can we at least still `log.Warn` the error so it's brought to the surface?

also, if there was an error, `out` will be an empty `[]byte` slice. will callers expect that?